### DISCUSSION TITLE
ignoreFiles in config.yaml

### DISCUSSION THREAD

**Discussion by AntoineSoetewey** [jpanther/congo#58] [Q&A] [answered]
Hello,

When I run `blogdown:::serve_site()` (following [Alison Hill](https://www.apreshill.com/blog/2020-12-blogdown-checks/)'s advice):

![Screenshot 2021-12-21 at 11 56 54](https://user-images.githubusercontent.com/17910063/146919331-b24153c7-60d3-4b94-b3d6-2296298dce61.png)

It says that I should add `ignoreFiles = ["\\.Rmd$", "\\.Rmarkdown$", "_cache$"]` to my config.yaml file, like this:

![Screenshot 2021-12-21 at 11 57 27](https://user-images.githubusercontent.com/17910063/146919525-f382bb08-44a2-46c9-abc4-393f261e478a.png)

When added, here is what I have when I run `blogdown:::serve_site()` again:

![Screenshot 2021-12-21 at 11 57 23](https://user-images.githubusercontent.com/17910063/146919601-0f4dcd53-6e91-400d-80ab-9694614829a0.png)

And I cannot serve the site anymore.

(My repo is https://github.com/AntoineSoetewey/antoinesoetewey.com).

Any idea how I could fix this?

Regards,
Antoine

**Comment by jpanther** [comment, marked as answer]
I'm not familiar with blogdown but that looks to me like you're trying to use TOML syntax inside a YAML formatted file.

**Reply by AntoineSoetewey** [reply]
Thanks for your reply.

I have also added it to the config.toml file, but that does not clear the point in the to do list:

![Screenshot 2021-12-22 at 08 41 17](https://user-images.githubusercontent.com/17910063/147054501-3a2f54ea-9151-41c7-98e9-8018cce9fe52.png)

I'll ask on the blogdown repo, thanks anyway !

**Reply by AntoineSoetewey** [reply]
After seeing this [tweet](https://twitter.com/andrew_leach/status/1334251603365281794/photo/1), here is how the ignoreFiles attribute should be in order to clear the point in the to do list:

![Screenshot 2021-12-22 at 08 47 41](https://user-images.githubusercontent.com/17910063/147055426-7c15c900-d239-4281-a8bf-66eee10b22b4.png)
